README.md

```diff
@@ -49,7 +49,8 @@ A lightweight and fast control to render a select component that can display hie
   - [noMatchesText](#nomatchestext)
   - [keepTreeOnSearch](#keeptreeonsearch)
   - [keepChildrenOnSearch](#keepchildrenonsearch)
-  - [simpleSelect](#simpleselect)
+  - [keepOpenOnSelect](#keepopenonselect)
+  - [mode](#mode)
```

Comment:
This is a breaking change and most likely belongs to bundling props PR. I thought we're merging this without needing any breaking changes?